src/IssueViz.Security.UnitTests/Taint/TaintList/TaintIssuesControlViewModelTests.Caption.cs

```diff
@@ -120,6 +121,46 @@ public void ActiveDocChanged_ExpectedCaptionSet()
             testSubject.WindowCaption.Should().Be(DefaultCaption);
         }
 
+        [TestMethod]
```

Comment:
These tests are separate from TaintIssuesControlViewModelTests because they test a lot more things: they test that the filtering and the caption calculation are working properly (you should see issues only for the selected document). This is not a functionality that is required from the server type, so these tests are unnecessarily complicated.  

I think it's better to put simple tests in TaintIssuesControlViewModelTests, one for ctor and one for document changed (and it doesn't even matter what the document is and if there are any issues, since the ServerType property is standalone).